plugins/coverbrowser.koplugin/listmenu.lua

```diff
@@ -541,10 +541,12 @@ function ListMenuItem:update()
 end
 
 function ListMenuItem:paintTo(bb, x, y)
-    -- We may get non-integer x or y (see mosaicmenu.lua)
-    -- Make them integer:
-    x = math.floor(x)
-    y = math.floor(y)
+    -- We used to get non-integer x or y that would cause some mess with image
+    -- inside FrameContainer were image would be drawn on top of the top border...
+    -- Fixed by having TextWidget:updateSize() math.ceil()'ing its length and height
+    -- But let us know if that happen again
+    assert(x == math.floor(x))
```

Comment:
It looks like the menu should remain accessible, so I'd say something ugly on the screen is preferable to what would look like a crash.

And no, but is that something https://github.com/koreader/koreader/blob/a8a7ebe1d4e7d2747cda8cb15dd0020e874c01ce/frontend/logger.lua#L9 should do? :-P